src/PowerShellEditorServices/Services/TextDocument/Handlers/FoldingRangeHandler.cs

```diff
@@ -36,19 +36,21 @@ public override Task<Container<FoldingRange>> Handle(FoldingRangeRequestParam re
         {
             if (cancellationToken.IsCancellationRequested)
             {
-                _logger.LogDebug("FoldingRange request canceled for file: {0}", request.TextDocument.Uri);
+                _logger.LogDebug("FoldingRange request canceled for file: {uri}", request.TextDocument.Uri);
```

Comment:
If only we had tooling that could just fix all of these for us. I personally even prefer the simpler `$"foo {var}"` style.

Reply:
I'm using vscode with the rosyln linter, I think there are code actions to fix this stuff.

Reply:
That'd be lovely.